packages/frontend/src/components/Nav.tsx

```diff
@@ -128,15 +204,42 @@ const Nav: React.FC = () => {
   const { track } = useAmplitude()
 
   const [anchorEl, setAnchorEl] = React.useState(null)
+  const [showBanner, setShowBanner] = useState(false)
+  const [modalOpen, setModalOpen] = useState(false)
+  const handleModalToggle = () => {
+    setModalOpen(!modalOpen)
+  }
   const handleClick = (event: any) => {
     setAnchorEl(event.currentTarget)
   }
   const handleClose = () => {
     setAnchorEl(null)
   }
 
+  useEffect(() => {
+    if (userLocation === 'GB') {
+      // 'GB' is the country code for the United Kingdom
+      setShowBanner(true)
+    } else {
+      setShowBanner(false)
+    }
+  }, [userLocation])
+
   return (
     <div className={classes.root}>
+      {showBanner && (
```

Comment:
can't read much on smaller devices. think we should unwrap and show the entire content like the one below.

<img width="214" alt="Screenshot 2024-07-29 at 5 20 23 PM" src="https://github.com/user-attachments/assets/4fdc0364-46aa-4e41-b784-8c43cfcdd0c3">

Reply:
this is how it looks on uniswap for mobile devices

<img width="213" alt="Screenshot 2024-07-29 at 5 22 28 PM" src="https://github.com/user-attachments/assets/d5230b4b-453c-43ad-a262-990234c26f35">